packages/augur-core/source/contracts/reporting/Universe.sol

```diff
@@ -397,6 +397,7 @@ contract Universe is IUniverse, CashSender {
         withdraw(address(this), _cashBalance, msg.sender);
         openInterestInAttoCash = openInterestInAttoCash.sub(_marketOI);
         cash.approve(address(augur), _cashBalance);
+        vat.hope(address(augur));
```

Comment:
Should we do the `hope` in the the initialization of universe or at the point of forking, since it doesn't need to happen for every market that moves?

Reply:
I believe the `approve` is here so the approval is limited to the OI owned by this market.  This offers some protection if a malicious fork can somehow manage to steal all of the assets in Augur, it would be limited to being able to steal whatever assets it was approved for.  Vat DAI on the other hand cannot limit the losses to the approved amount since approval is all-or-nothing.  So the question I think is whether or not we want to split them up (which would be confusing) or keep them together (which results in unnecessary calls to vat.hope).

Reply:
It's called relatively infrequently (Only during a fork but for all unfinalized markets). That said it probably should go in initialization